src/main/groovy/org/pih/warehouse/core/Constants.groovy

```diff
@@ -91,7 +91,6 @@ class Constants {
     static final String PRODUCT_INVENTORY_TRANSACTION_TYPE_ID = "11"
     static final String CYCLE_COUNT_PRODUCT_INVENTORY_TRANSACTION_TYPE_ID = "12"
     static final String CYCLE_COUNT_ADJUSTMENT_TRANSACTION_TYPE_ID = "13"
```

Comment:
Do we need the distinction between a product inventory transaction and cycle count product inventory transaction? Same with the adjustment?